docs/DeveloperGuide.md

```diff
@@ -689,18 +695,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
    Use case ends.
 
 **Extensions**
-* 2a. The update data is invalid.
+* 2a. The guest's bill is negative.
```

Comment:
Should we say "The guest's updated bill is negative"

Reply:
Agreed!